### DISCUSSION TITLE
List / comparison of similar projects

### DISCUSSION THREAD

**Discussion by deliciouslytyped** [xesscorp/skidl#210] [General]
Is there a list of/comparison of similar projects/tools somewhere?

For example, I ran into https://github.com/BerkeleyHCI/PolymorphicBlocks / https://pypi.org/project/edg/  (and https://github.com/BerkeleyHCI/edg-ide), but I didn't have time to look into how it compares, or why it doesn't cite skidl.

Then there are things like https://github.com/michaelgale/pcbflow

**Comment by devbisme** [comment]
Here's a list of SKiDL-like Github projects I found using [this](https://devbisme.github.io/RepoRecon/?topic=kicad&filter=&sort=pushed:desc). These all mentioned KiCad in some way, so projects targeting other EDA tools would be missed. Many projects in this list are old and dead. 

* https://github.com/chmod775/trace
* https://github.com/google/pcbdl
* https://github.com/bristinemuluh/piLang
* https://github.com/jdisanti/netmuncher
* https://github.com/baryluk/edag
* https://github.com/Blinkinlabs/circuitpainter
* https://github.com/esophagoose/earthground
* https://github.com/Monacrylic/KiCAD-schematic-utils
* https://github.com/ydnatag/pypcb
* https://github.com/paulo-raca/PyPcb
* https://github.com/iemxblog/hpcb

Finally, here's the one that I found and used for guidance when I started SKiDL: [PHDL](https://phdl.sourceforge.net/1.0/index.php).

**Comment by deliciouslytyped** [comment]
When I tried searching Google, I think I used something along the lines of "python pcb hdl".

**Comment by devbisme** [comment]
I found a few more:
* https://github.com/tscircuit/tscircuit
* https://github.com/atopile/atopile
